ui/ui-components/components/destination/DestinationSampleRecord.tsx

```diff
@@ -2,10 +2,11 @@ import { useCallback, useMemo, useState } from "react";
 import SampleRecordCard, {
   SampleRecordCardProps,
 } from "../record/SampleRecordCard";
-import { Actions, Models } from "../../utils/apiData";
+import { Models } from "../../utils/apiData";
 import { useDebouncedValue } from "../../hooks/useDebouncedValue";
 import { useApi } from "../../contexts/api";
 import { useGrouparooModel } from "../../contexts/grouparooModel";
+import { DestinationRecordPreview } from "@grouparoo/core/src/actions/destinations";
```

Comment:
For now, please continue to use the `ui-components/utils/apiData.ts` loading mechanism for now - there were some problems with node 12 (I think?) and needing to forcibly re-export only the types... it might be fixed now, but we should upgrade that holistically at once.